spec.html

```diff
@@ -58,7 +58,7 @@ <h1>Runtime Semantics: HostResolveImportedModule ( _referencingScriptOrModule_,
         <ins>
           If _assertions_ has an entry _entry_ such that _entry_.[[Key]] is *"type"*, let _type_ be _entry_.[[Value]]. The following requirements apply:
           <ul>
-            <li><ins>If _type_ is *"json"*, then this algorithm must either invoke ParseJSONModule and return the resulting Module Record, or throw an exception.</ins></li>
+            <li><ins>If _type_ is *"json"*, then this algorithm must either invoke ParseJSONModule and, unless ParseJSONModule resulted in an abrupt completion, return the resulting Module Record, or throw an exception.</ins></li>
```

Comment:
```suggestion
            <li><ins>If _type_ is *"json"*, then this algorithm must either invoke ParseJSONModule and return the resulting Completion Record, or throw an exception.</ins></li>
```
? that way it encapsulates "whatever ParseJSONModule can do" which is either a Module Record or an AbruptCompletion

Reply:
sounds better, thanks!